.test-infra/jenkins/job_LoadTests_Java.groovy

```diff
@@ -69,6 +53,53 @@ for (testConfiguration in testsConfigurations) {
     ) {
         description(testConfiguration.jobDescription)
         commonJobProperties.setTopLevelMainJobProperties(delegate)
-        loadTestsBuilder.buildTest(delegate, testConfiguration.jobDescription, testConfiguration.runner, testConfiguration.jobProperties, testConfiguration.itClass)
+        loadTestsBuilder.loadTest(delegate, testConfiguration.jobDescription, testConfiguration.runner, testConfiguration.jobProperties, testConfiguration.itClass)
+    }
+}
+
+def smokeTestConfigurations = [
+        [
+                title        : 'GroupByKey load test Direct',
+                itClass      : 'org.apache.beam.sdk.loadtests.GroupByKeyLoadTest',
+                runner       : CommonTestProperties.Runner.DIRECT,
+                jobProperties: [
+                        publishToBigQuery: true,
+                        bigQueryDataset  : 'load_test_SMOKE',
+                        bigQueryTable    : 'direct_gbk',
+                        sourceOptions    : '{"numRecords":100000,"splitPointFrequencyRecords":1}',
+                        stepOptions      : '{"outputRecordsPerInputRecord":1,"preservesInputKeyDistribution":true}',
+                        fanout           : 10,
+                        iterations       : 1,
+                ]
+        ],
+        [
+                title        : 'GroupByKey load test Dataflow',
```

Comment:
I added a tiny variant of a Dataflow test too in order to quickly detect if nothing gets broken on dataflow too (dependencies, worker problems etc). In the future, other runners should have their variants too. 

Do you think it might be useful to add such smoke suite to post-commit jobs?

Reply:
I think adding smoke tests for other runners is probably useful. Most have local runners so the test should be fairly quick. Dataflow doesn't have this option and takes a few minutes to start up, so tests less than a few minutes will be mostly startup time.

Reply:
Good idea - I added suites for Spark and Flink.